api/api.py

```diff
@@ -484,3 +485,119 @@ def expense_by_country(self, request, pk=None):
             "category": categories,
             "data": agregateExpenses
         })
+
+    @action(detail=False)
+    def lupo(self, request, pk=None):
+        a = Expense.objects.filter(budget__country__name="Angola", budget__year=2019, level=0).aggregate(
+            total=Sum("budget_aggregated"))
+        return Response(a["total"])
+
+    @action(detail=False)
```

Comment:
Leave only official code on PR to develop or master. You can put your test codes into a particular branch without push it into main repository.